### ISSUE TITLE
[Term Entry] AI Backpropagation

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#2134]
### Reason/inspiration (optional)

We would like a new term entry for the concept of Backpropagation.

The entry should go in a new file in docs/content/ai/concepts/neural-networks/terms/backpropagation/backpropagation.md

The entry should
- Describe what Backpropagation is.
- Explain how it is used in an Artificial Neural Network.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

**Comment by Himasnhu-AT** [comment]
@codecademy-docs Hi, I would like to contribute to this. Please assign it to me.

**Comment by KTom101** [comment]
@Himasnhu-AT You're assigned! You can check out the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) for additional information on contributing. See [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) for an overview of styling tips that the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) covers in detail. After creating a PR, maintainer(s) (with the collaborator label) will add in comments/suggestions to address any revisions prior to approving.

Is this your first contribution to Docs? If so, we're curious to know how you found out about contributing to Docs.

**Comment by Himasnhu-AT** [comment]
@codecademy-docs Sent the PR. Let me know how it is.
